tests/schema/tests.py

```diff
@@ -76,14 +75,13 @@ def tearDown(self):
     def delete_tables(self):
         "Deletes all model tables for our models for a clean test environment"
         converter = connection.introspection.table_name_converter
-        with atomic():
+        with connection.schema_editor() as editor:
```

Comment:
Same thing here, `atomic()` will only be used if `connection.features.can_rollback_ddl`.